client/api/volume/v1beta1/api.proto

```diff
@@ -24,6 +24,8 @@ service Volume {
     rpc GetVolumeDiskNumber(VolumeDiskNumberRequest) returns (VolumeDiskNumberResponse) {}
     // GetVolumeIDFromMount gets the volume id for a given mount
     rpc GetVolumeIDFromMount(VolumeIDFromMountRequest) returns (VolumeIDFromMountResponse) {}
+    // WriteVolumeCache write volume cache to disk
+    rpc WriteVolumeCache(WriteVolumeCacheRequest) returns (WriteVolumeCacheResponse) {}
```

Comment:
I think we need a v1beta2 if we're going to modify the API since csi-proxy versions are not tied to k8s versions

Reply:
updated.